src/components/Layout/Header/Header.jsx

```diff
@@ -16,13 +18,13 @@ export function Header() {
 		<header className={styles.header}>
 			<div className="__container">
 				<div className={styles.logo}>
```

Comment:
You should better replace <div> with Link, so clickable area would be wider
```suggestion
				<Link to="/" className={styles.logo}>
				     Proved<span>Code</span>
				</Link>
```